build.gradle

```diff
@@ -3925,6 +3926,7 @@ allprojects {
         project.compileShimsJava {
             options.compilerArgs.addAll([
                 '-implicit:none',
+                '--module-version', "$RELEASE_VERSION_SHORT",
```

Comment:
I'm not sure this is needed (although it doesn't hurt), since the shims are only used for testing and are not included in any jar file.

Reply:
Seeing version information in stack traces of failed test can be helpful. ;-)

Reply:
OK, that seems reasonable, and I can't think of any downside.

Reply:
The only thing that makes me slightly nervous is seeing that this is needed on a number of places, leading to more code duplication (and the build.gradle is already rather large). But that is not a problem introduced with this PR, so I'm not requesting a change for this.

Reply:
Yeah. At least, the four duplicated places look very similar to each other. Perhaps, those lines (adding module-related compiler args) can be extracted into a dedicated method. But that should be done, if at all, in a different task/PR.